### ISSUE TITLE
Windows: Get identical ActionId for different handlers under same trigger when using old trigger API

### ISSUE THREAD

**Issue by wenchunzhang** [TIBCOSoftware/flogo-lib#236]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[] enhancement request
[] feature request
[] support request
[] general question
```

**Current behavior (how does the issue manifest):**
Old trigger API, multiple handlers under same trigger, the ActionId for each handler is same

**Expected behavior:**
Different handler should have different actionId 

**Minimal steps to reproduce the problem (not required if feature enhancement):**
Reproduced this with Trigger(old trigger API), and in trigger having multiple handers defined.

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment (Operating system, docker version, browser & verison if webui, etc):**
Window executable

**Flogo version (CLI & contrib/lib. If unknown, leave empty or state unknown):** 0.X.X
0.5.6

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by mellistibco** [comment]
@weching you should look to upgrade to the newest API, as we'll be deprecating the legacy APIs at some point.

**Comment by weching** [comment]
@mellistibco 
Upgrade to use new API definitely an alternative way. 

But the reason for this is: ActionId being set as time nanosecond in the loop execution for handlers, since on Windows there's no precise resolution for nanosecond and loop get execute too fast, so id get same. 

```
     hc.ActionId = strconv.Itoa(time.Now().Nanosecond())

```

We could genera UUID instead of time nano.  Would you want to leave the legacy API not fix?

**Comment by lixingwang** [comment]
Has been fixed at PR #240
